### PR TITLE
10.0 mig base view inheritance extension

### PR DESCRIPTION
**Pull request by lfreeke**
Migration from 9 to 10

### CODE REVIEW

**Comment by lfreeke** [comment]
@hbrunn ping

File: base_view_inheritance_extension/README.rst [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+This module was written to make it simple to add custom operators for view
+inheritance.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+    :alt: Try me on Runbot
+    :target: https://runbot.odoo-community.org/runbot/149/9.0
```
Comment (hbrunn) [inline base_view_inheritance_extension/README.rst]: 10.0

File: base_view_inheritance_extension/__manifest__.py [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 {
     "name": "Extended view inheritance",
-    "version": "9.0.1.1.0",
+    "version": "10.0.1.1.0",
```
Comment (MiquelRForgeFlow) [inline base_view_inheritance_extension/__manifest__.py]: 10.0.1.0.0

**Review by hbrunn — APPROVED** [review]
small nit, rest looks good

File: base_view_inheritance_extension/models/ir_ui_view.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        <$node position="attributes">
+            <attribute name="$attribute" operation="python_dict" key="$key">
+                $keyvalue
+            </attribute>
+        </$node>"""
+        node = self.locate_node(source, specs)
+        for attribute_node in specs:
+            python_dict = tools.safe_eval(
+                node.get(attribute_node.get('name')) or '{}',
+                UnquoteEvalObjectContext()
```
Comment (hbrunn) [inline base_view_inheritance_extension/models/ir_ui_view.py:105]: this is where travis' failure comes from, add parameter `nocopy=True`, that should fix the test run

**Review by hbrunn — CHANGES_REQUESTED** [review]
travis fails

**Comment by MiquelRForgeFlow** [comment]
@lfreeke please, rebase

**Comment by lfreeke** [comment]
@mreficent I rebased.

**Review by hbrunn — APPROVED** [review]

File: base_view_inheritance_extension/models/ir_ui_view.py (hunk 3 above)
Comment (MiquelRForgeFlow) [reply base_view_inheritance_extension/models/ir_ui_view.py:105]: it gives a warning, not a failure, in runbot full logs

@lfreeke, could you attend @hbrunn comment?

**Comment by hbrunn** [comment]
my comment was wrong, so there's nothing to do. [Edit: no, it's not. still todo]

**Pull request merged**